ui/apps/platform/src/routePaths.ts

```diff
@@ -247,6 +249,10 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
         featureFlagRequirements: allEnabled(['ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL']),
         resourceAccessRequirements: everyResource(['Administration']),
     },
+    'exception-management': {
+        featureFlagRequirements: allEnabled(['ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL']),
+        resourceAccessRequirements: everyResource(['Administration']),
```

Comment:
I think the minimum required resource for this page should be `VulnerabilityManagementRequests`. A user who opens a deferral/FP should be able to view this page, and they might not have access to the `Administration` resource. We might even need to distinguish between READ_ACCESS and READ_WRITE_ACCESS for regular requester level users inside this page.

Once we get into the page itself, approval based features will also require conditional rendering depending on the user's access to `VulnerabilityManagementApprovals`.

Reply:
If a user has access to `VulnerabilityManagementApprovals`, that should imply they have access to `VulnerabilityManagementRequests` as well, right? Does it automatically give access to `VulnerabilityManagementRequests` though or will we need to do an `OR` for both of those permissions?

Reply:
Hmm that I'm not sure. We would have to check with the BE team to verify. My gut feeling is you are right, that a user should have either `OR` permission to access the page.

It certainly seems possible that someone would configure a user with `VulnerabilityManagementApprovals`, but not `VulnerabilityManagementRequests`, so it makes sense to check either one for the top level routing. We will need to confirm what the API behavior is though to handle how the UI is displayed inside the page.

Reply:
I'll add both (with an OR) for now and maybe ask about it in our VM sync tomorrow.